apps/web/modules/users/views/users-public-view.getServerSideProps.tsx

```diff
@@ -104,11 +104,13 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
 
   if (isDynamicGroup) {
     const destinationUrl = `/${usernameList.join("+")}/dynamic`;
+    const originalQueryString = new URLSearchParams(context.query as Record<string, string>).toString();
```

Comment:
I would expect that all `user` values would be removed from this query string, because that is already extracted earlier and used as `usernameList`.